### ISSUE TITLE
Remove role=alert from MUI alerts that are only used as additional info, not time-sensitive info

### ISSUE THREAD

**Issue by SiniCode** [UniversityOfHelsinkiCS/palaute#1675]
Role 'alert' makes screen reader announce the info quickly and it's for time-sensitive information like session expiration warning etc. The Alert components have this role built-in but it should be removed if the component is only used for styling info boxes.
